ui/widget/src/main/java/de/danoeh/antennapod/ui/widget/WidgetUpdater.java

```diff
@@ -190,8 +161,71 @@ public static void updateWidget(Context context, WidgetState widgetState) {
                 views.setInt(R.id.butPlay, "setVisibility", View.VISIBLE);
             }
 
-            int backgroundColor = prefs.getInt(PlayerWidget.KEY_WIDGET_COLOR + id, PlayerWidget.DEFAULT_COLOR);
-            views.setInt(R.id.widgetLayout, "setBackgroundColor", backgroundColor);
+            Bitmap icon;
+            RequestOptions option = new RequestOptions()
+                    .dontAnimate()
+                    .transform(new FitCenter());
+
+            if (showCoverAsBcg) {
+                views.setViewVisibility(R.id.imgvCover, View.GONE);
+                views.setViewVisibility(R.id.imgvBackground, View.VISIBLE);
+                views.setViewVisibility(R.id.imgvBlurOverlay, View.VISIBLE);
```

Comment:
Blurring works without overlay views using the Glide `new RequestOptions().transform(new FastBlurTransformation())`